sdk/cosmos/azure-cosmos/pom.xml

```diff
@@ -366,6 +366,27 @@ Licensed under the MIT License.
         </plugins>
       </build>
     </profile>
+    <profile>
+      <!-- integration tests, requires Cosmos DB endpoint with multi region support -->
+      <id>multi-region</id>
```

Comment:
some of the CIs provide multi-region account to the existing test group.
Why do we need this additional test group? 
also is there a specific consistency configured in the CI for the test group?

Reply:
We don't have any group which runs only multi-region single master. My test need that case explicitly